dev/com.ibm.ws.security.jwt/src/com/ibm/ws/security/jwt/config/JweConsumerConfig.java

```diff
@@ -0,0 +1,27 @@
+/*******************************************************************************
```

Comment:
It appears this interface is not used yet. The code could be simplified by removing this interface and adding the methods back to the JwtConsumerConfig interface and make it the authority of all things related to JWT consumption, including JWE.